### PR TITLE
AT_04.06 _003| Breadcrumbs | Builds for user page > Verify builds list's sorted in ascending order by default

### PR DESCRIPTION
**Pull request by ecatherrina**
https://trello.com/c/0g2ASF5c/964-at0406-003-breadcrumbs-builds-for-user-page-verify-builds-lists-sorted-in-ascending-order-by-default

### CODE REVIEW

File: cypress/e2e/breadcrumbsBuilds.cy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+describe('Breadcrumbs | Builds for user page', () => {
```
Comment (Rom-T) [inline cypress/e2e/breadcrumbsBuilds.cy.js:3]: Please name the describe as the name of your file.

File: cypress/e2e/breadcrumbsBuilds.cy.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    beforeEach('', () => {
+        cy.get('a[href^="newJob"]').click();
+        cy.get('input#name').type('Project');
```
Comment (Rom-T) [inline cypress/e2e/breadcrumbsBuilds.cy.js:7]: Please move `'Project'` in the fixtures

File: cypress/e2e/breadcrumbsBuilds.cy.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        cy.get('a[href$=builds]').click();
+    
+        });
+        
+    
+    it('AT_04.06 _003 | Verify builds list is sorted in ascending order by default', () => {
+            
+        cy.get('#projectStatus th:nth-child(2) a').click();
+    
+        cy.get('th:nth-child(2) .sortarrow').should('contain', '↑');
```
Comment (Rom-T) [inline cypress/e2e/breadcrumbsBuilds.cy.js:25]: Please move `'↑'` in the fixtures

File: cypress/e2e/breadcrumbsBuilds.cy.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        cy.get('#projectStatus tbody>tr:nth-child(odd)').should('contain', '#2');
+        cy.get('#projectStatus tbody>tr:nth-child(even)').should('contain', '#1');
```
Comment (Rom-T) [inline cypress/e2e/breadcrumbsBuilds.cy.js:27]: Please move `'#2'` and `'#1'` in the fixtures

Comment (ecatherrina) [reply cypress/e2e/breadcrumbsBuilds.cy.js]: Done

**Review by MarinaMikhailova — APPROVED** [review]

**Review by alenakhan — APPROVED** [review]

**Review by kulmashev — APPROVED** [review]

**Review by kulmashev — APPROVED** [review]

**Review by NinaKosova — APPROVED** [review]
lgtm

**Pull request merged**